docs/products/opensearch/howto/opensearch_alerting.rst

```diff
@@ -0,0 +1,142 @@
+
+Create using Dashboards UI
+**************************
+
+In order to create an alert via OpenSearch Dashboards interface, follow these steps:
+
+Log in Dashboard interface
+***************************
+
+1. Log in to the `Aiven web console <https://console.aiven.io>`_ and select your OpenSearch service.
+
+2. Click the **Overview** tab -> **OpenSearch Dashboards** under `Connection Information`
+
+   This opens the OpenSearch Dashboard
+
+3. In OpenSearch Dashboard Click  on top left to open the left side panel -> Under `OpenSearch Plugins` -> **Alerting**
+
+.. note::
+    To configure each alert there are ``Monitor``, ``Data source``, ``Query``, ``Trigger`` and ``Destination`` needs to be created.
+
+
+Create a destination
+********************
+
+1. Click the **Destination** tab -> **Add destination**
+   
+2. **Name** -> ``slack-test``
+
+   **Type** -> ``Slack``
+
+   **Settings** **Webhook URL:** -> ``https://your_slack_webhook_URL``
+
+.. note::
+   Destination Type can be: ``Amazon Chime``, ``Slack``, ``Custom webhook`` or ``Email``
+
+.. important::
+   When using email you need to have a SMTP server configured for a valid domain to deliver email notifications
+
+Create a monitor
+****************
+
+1. Click the **Monitors** tab -> **Create monitor**
+
+2. **Monitor details**
+   
+   **Monitor name** -> ``High CPU Monitor``
+
+   **Monitor name** -> ``Per query monitor``
+   
+   **Monitor defining method** -> ``Visual editor`` 
+
+   **Schedule** **Frequency** -> ``By interval``
+
+   **Run every** -> ``1`` ``Minutes``
+
+.. note::
+   Schedule Frequency can be `By internal`, `Daily`, `Weekly`, `Monthly`, `Custom CRON expression`
+
+3. **Data source** 
+ 
+   **index** -> ``sample-host-health``
+
+   **Time field** -> ``timestamp``
+
+4. **Query**
+
+   **Metrics** -> **Add metric** 
+
+   **Aggregation** ``avrerage()`` **Field** ``cpu_usage_percentage``
+
+   **Time range for the last** ``3`` ``minutes``
+
+Create a trigger
+****************
+
+1. **Add trigger**
+
+   **Trigger name** -> ``high_cpu``
+
+   **Severity level** -> ``1 (Highest)``
+
+   **Trigger condition** ``IS ABOVE`` ``75``
+
+.. note::
+   You can see a visual graph below trigger with the index data and the trigger condition you have defined as a red line
+
+2. **Actions**
+     
+   **Action name** -> ``slack``
+
+   **Destination** -> ``slack-test``
+
+   **Message subject** -> ``High CPU Test Alert``
+
+.. note::
+   Multiple Actions can be defined, in this example we will define one action to send notification to destination we have defined in step 4
+
+Alert Message
+*************
+
+**Message** can be adjusted as needed, check **Message Preview** to see the sample and use **Send test message** to validate notification delivery
+
+Click on **Create** and your monitor is ready!
+
+* For further details on `alerting monitors configuration <https://opensearch.org/docs/latest/monitoring-plugins/alerting/monitors/>`_
+
+.. _Code:
+
+Create programmatically
```

Comment:
Let's make this a separate article "Create an alert with OpenSearch Dashboards programmatically" and give more words and an explanation of how to know what JSON goes where, linking to their docs if appropriate. This article can then also signpost that other article.